packages/rest/src/controllers/basic-messages/BasicMessageController.ts

```diff
@@ -9,17 +9,16 @@ import {
   Param,
   Post,
 } from 'routing-controllers'
-import { Inject, Service } from 'typedi'
+import { inject, injectable } from 'tsyringe'
 
 import { BasicMessageRequest } from '../../schemas/BasicMessageRequest'
 
 @JsonController('/basic-messages')
-@Service()
+@injectable()
 export class BasicMessageController {
-  @Inject()
   private agent: Agent
 
-  public constructor(agent: Agent) {
+  public constructor(@inject('agent') agent: Agent) {
```

Comment:
`@inject` is only needed for interfaces. With classes you can leave the `@inject` out